packages/opera.rb

```diff
@@ -5,32 +5,30 @@ class Opera < Package
   homepage 'https://www.opera.com/'
   version `curl -Ls https://get.geo.opera.com/pub/opera/desktop/ | grep 'a' | tail -1 | cut -d '"' -f 2 | sed -e 's|\/||' | perl -e 'while(<>) { $_ =~ s/[\r\n]//g; print "$_" }'`
   compatibility 'x86_64'
-  case ARCH
-  when 'x86_64'
-    source_url 'file:///dev/null'
-    source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
-    depends_on 'alien' => :build
-    depends_on 'gtk3'
-    depends_on 'gsettings_desktop_schemas'
-    depends_on 'harfbuzz'
-    depends_on 'graphite'
-    depends_on 'sommelier'
-  end
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  depends_on 'alien' => :build
+  depends_on 'gtk3'
+  depends_on 'gsettings_desktop_schemas'
+  depends_on 'harfbuzz'
+  depends_on 'graphite'
+  depends_on 'sommelier'
```

Comment:
We need the `case...end` block since this package can only be installed on x86_64.  Otherwise, all the dependencies are installed for incompatible architectures.

Reply:
Done